flink-core/src/main/java/org/apache/flink/configuration/ConfigConstants.java

```diff
@@ -487,6 +487,11 @@
 	public static final String MESOS_INITIAL_TASKS = "mesos.initial-tasks";
 
 	/**
+	* Constraints for placement of Mesos tasks.
+	*/
+	public static final String MESOS_CONSTRAINTS = "mesos.constraints.hard.hostattribute";
```

Comment:
Please remove this constant, since you're using the new ConfigOption approach which doesn't necessitate an entry here.